src/main/java/javax/measure/unit/NonSI.java

```diff
@@ -584,7 +584,8 @@ public static NonSI getInstance() {
      * A unit of pressure equal to <code>100 kPa</code>
      * (standard name <code>bar</code>).
      */
-    public static final Unit<Pressure> BAR = nonSI(PASCAL.times(100000));
+    public static final Unit<Pressure> DEKAMICROBAR = new AlternateUnit<>("dubar", NEWTON.divide(METRE.pow(2)));
+    public static final Unit<Pressure> BAR = nonSI(DEKAMICROBAR.times(100000));
```

Comment:
I suggest adding a unit test to `UnitTest.java` to verify this. Perhaps something like:
```
@Test
public void bar() {
    assertThat(NonSI.BAR.getConverterTo(SI.PASCAL).convert(1.0)).isEqualTo(100_000.0);
    assertThat(SI.PASCAL.getConverterTo(NonSI.BAR).convert(100_000.0)).isEqualTo(1.0);
    assertThat(NonSI.BAR.toString()).isEqualTo("bar");
    assertThat(Unit.valueOf("bar")).isEqualTo(NonSI.BAR);
    assertThat(Unit.valueOf("dµbar")).as("Regrettable that we need to expose this odd unit").isNotNull();
}
```

Reply:
Done.